app/assets/stylesheets/petitions/views/_sign.scss

```diff
@@ -6,3 +6,7 @@
   padding-left: $gutter * 1.5;
   margin-bottom: $gutter * 1.3;
 }
+
+#signature_postcode, #petition_creator_signature_postcode {
+  text-transform: uppercase;
+}
```

Comment:
Are you having to do this because the data hasn't been sanitized properly? We should normalise the formatting of postcodes on the server.